docs/nydusd.md

```diff
@@ -265,7 +265,10 @@ Currently, the mirror mode is only tested in the registry backend, and in theory
               // When Dragonfly does not cache data, nydusd will pull it from "X-Dragonfly-Registry".
               // If not set "X-Dragonfly-Registry", Dragonfly will pull data from proxy.registryMirror.url.
               "X-Dragonfly-Registry": "https://index.docker.io"
-            }
+            },
+            // Endpoint of mirror server health checking (including scheme).
+            // Use host/v2 if not configure this item.
+            "ping_url": "http://127.0.0.1:40901/server/ping",
```

Comment:
Maybe `health_check_interval` and `failure_limit` need also to be explained here?

Reply:
Yes, I have added it.